pkgs/top-level/all-packages.nix

```diff
@@ -12311,6 +12311,14 @@ with pkgs;
   geany = callPackage ../applications/editors/geany { };
   geany-with-vte = callPackage ../applications/editors/geany/with-vte.nix { };
 
+  gimagereader-gtk = callPackage ../by-name/gi/gImageReader/package.nix {
+    withQt6 = false;
+  };
+
+  gimagereader-qt = qt6Packages.callPackage ../by-name/gi/gImageReader/package.nix {
+    withQt6 = true;
+  };
```

Comment:
it's a bit odd to have gImageReader then gimagereader-gtk / gimagereader-qt as attribute names. 
gImageReader-gtk / gImageReader-qt would be more consistant. Also `gImageReader-gtk` is the same as `gImageReader` so just ditch it.